publishing-bot/manifests/storage-class.yaml

```diff
@@ -0,0 +1,7 @@
+kind: StorageClass
```

Comment:
This should go ... somewhere else.  Probably in `clusters/kubernetes-public/aaa` - see https://www.terraform.io/docs/providers/kubernetes/r/storage_class.html

Reply:
This could be a PR of its own